src/main/target/STM32H750/target.h

```diff
@@ -18,16 +18,51 @@
  * If not, see <http://www.gnu.org/licenses/>.
  */
 
+/*
+ * This target won't actually build a target that you can boot, it is meant as
+ * a base target ONLY.
+ *
+ * When defining a target that uses this as a base you currently:
+ *
+ * 1) *must* define ALL the SPI instances that the target has, including their pins.
+ * 2) *must* define the storage subsystem used to boot, e.g. see CONFIG_IN_xxx.
+ * 3) *must* define all the settings required for the config storage system to
+ *    be able to load the config at boot time BEFORE it has actually loaded a config.
+ *    e.g. for QSPI define the QSPI instance, pins, mode, etc,
+ *         for SDCARD define the SD card bus (SPI/SDIO), pins, etc.
+ *
+ */
+
 #pragma once
 
+#ifndef TARGET_BOARD_IDENTIFIER
 #define TARGET_BOARD_IDENTIFIER "S750"
+#endif
 
+#ifndef USBD_PRODUCT_STRING
 #define USBD_PRODUCT_STRING     "Betaflight STM32H750"
+#endif
 
+#if !defined(USE_I2C)
+#define USE_I2C
 #define USE_I2C_DEVICE_1
 #define USE_I2C_DEVICE_2
 #define USE_I2C_DEVICE_3
 #define USE_I2C_DEVICE_4
+#define I2C_FULL_RECONFIGURABILITY
+#endif
+
+// Provide a default so that this target builds on the build server.
+#if !defined(USE_SPI)
```

Comment:
Don't pass these on the command line - and then this IF is unnecessary.

Reply:
the STM32H750 PR check build will fail without them, see the comment. I tried, that's what caused me to add the `#if` and force-push the branch again.  See the build logs from today and the git history of the PR.